src/patternfly/components/EmptyState/examples/EmptyState.md

```diff
@@ -191,12 +241,12 @@ cssPrefix: pf-c-empty-state
 | -- | -- | -- |
 | `.pf-c-empty-state` | `<div>` |  Initiates an empty state component. The empty state centers its content (`.pf-c-empty-state__content`) vertically and horizontally. **Required** |
 | `.pf-c-empty-state__content` | `<div>` |  Creates the content container. **Required** |
-| `.pf-c-empty-state__icon` | `<i>`, `<div>` |  Creates the empty state icon or icon container when used as a `<div>`. |
-| `.pf-c-title` | `<h1>, <h2>, <h3>, <h4>, <h5>, <h6>` |  Creates the empty state title. **Required** |
-| `.pf-c-empty-state__body` | `<div>` |  Creates the empty state body content. You can have more than one `.pf-c-empty-state__body` elements. |
-| `.pf-c-button.pf-m-primary` | `<button>` |  Creates the primary action button. |
-| `.pf-c-empty-state__primary` | `<div>` |  Container for primary actions. Can be used in lieu of using `.pf-c-button.pf-m-primary`. |
-| `.pf-c-empty-state__secondary` | `<div>` |  Container secondary actions. |
+| `.pf-c-empty-state__header` | `<div>` |  Creates the header container. **Required** |
+| `.pf-c-empty-state__title` | `<div>` |  Creates the empty state title container. |
+| `.pf-c-empty-state__icon` | `<div>` |  Creates the empty state icon container. |
+| `.pf-c-empty-state__body` | `<div>` |  Creates the empty state body content. There can be more than one `.pf-c-empty-state__body` elements. |
+| `.pf-c-empty-state__footer` | `<div>` |  Container for actions. **Required** |
+| `.pf-c-empty-state__actions` | `<div>` |  Container for actions. **Required** |
```

Comment:
Just a nit - these have the same description, which is accurate. I wonder if the footer should be more inline with the header and body desc? Like "Creates the footer container" - wdyt?